Sources/Sentry/include/SentryRateLimitCategory.h

```diff
@@ -1,5 +1,6 @@
 #import <Foundation/Foundation.h>
 
+static NSString * const SentryRateLimitCategoryDefault = @"default";
```

Comment:
Trying to point out here that we can never guarantee we list all categories here since the server might add new ones at anytime. The code needs to downgrade to `default` gracefully as it does